src/pudl/cli.py

```diff
@@ -94,40 +62,40 @@ def parse_command_line(argv):
 
 def main():
     """Parse command line and initialize PUDL DB."""
+    load_dotenv()
     args = parse_command_line(sys.argv)
 
     # Display logged output from the PUDL package:
     pudl.logging_helpers.configure_root_logger(
         logfile=args.logfile, loglevel=args.loglevel
     )
 
-    etl_settings = EtlSettings.from_yaml(args.settings_file)
-
-    pudl_settings = pudl.workspace.setup.derive_paths(
-        pudl_in=etl_settings.pudl_in, pudl_out=etl_settings.pudl_out
-    )
-    pudl_settings["sandbox"] = args.sandbox
-
-    bad_sqlite_version = version.parse(sqlite_version) < version.parse(
-        MINIMUM_SQLITE_VERSION
-    )
-    if bad_sqlite_version and not args.ignore_type_constraints:
-        args.ignore_type_constraints = False
-        logger.warning(
-            f"Found SQLite {sqlite_version} which is less than "
-            f"the minimum required version {MINIMUM_SQLITE_VERSION} "
-            "As a result, data type constraint checking will be disabled."
-        )
-
-    pudl.etl.etl(
-        etl_settings=etl_settings,
-        pudl_settings=pudl_settings,
-        clobber=args.clobber,
-        use_local_cache=not args.bypass_local_cache,
-        gcs_cache_path=args.gcs_cache_path,
-        check_foreign_keys=not args.ignore_foreign_key_constraints,
-        check_types=not args.ignore_type_constraints,
-        check_values=not args.ignore_value_constraints,
+    etl_job = etl.defs.get_job_def("etl_full")
+    etl_job.execute_in_process(
+        run_config={
+            "resources": {
+                "dataset_settings": {
+                    "config": EtlSettings.from_yaml(args.settings_file).datasets.dict(
+                        exclude={
```

Comment:
Yeah I think maybe we just remove them entirely, because I don't think there's any use for them if they're not being used for selecting subsets of tables. I can also add a method as suggested though if we want to keep the tables for some reason